KCL book, second edition (#29)
KittyCAD/kcl-book: kcl-book/src/reduce_geometry.md

Reviewer: 404 link

# Reduce and geometry

> **NOTE**: This chapter covers the old syntax for our previous sketch library (that didn't support constraint solvers). If you're new to KCL, you should probably skip this chapter and read the new chapter on sketch blocks. If you've already got old KCL code that used the old syntax for sketching, don't worry, you can keep using the old syntax, and this chapter will explain it. But new users shouldn't worry about this, unless they're working with old code.

Reviewer: ```suggestion
**WARNING**: This chapter covers the old syntax for our previous sketch library (that does not support constraint solvers). If you're new to KCL, you should probably skip this chapter and read the new chapter on sketch blocks. If you already have old KCL code that uses the old syntax for sketching, don't worry, you can keep using the old syntax, and this chapter explains it. But new users shouldn't worry about this, unless they're working with old code.
```
